app/views/cases/offender_sar/_case_details.html.slim

```diff
@@ -19,15 +19,17 @@
         = render partial: 'cases/offender_sar/subject_address', locals: { case_details: case_details }
         = render partial: 'cases/offender_sar/high_profile_subject', locals: { case_details: case_details }
         = render partial: 'cases/offender_sar/third_party', locals: { case_details: case_details }
-        - if case_details.third_party?
-          = render partial: 'cases/sar/third_party_relationship', locals: { case_details: case_details }
+        - unless case_details.third_party == false && case_details.recipient == 'subject_recipient'
```

Comment:
I wonder if it might be clearer to wrap this in a method that explains what this mean so we can have:

```ruby
unless data_subject_is_requesting_date_for_themselves?
```
I think this may make it easier to reason about 👍

Reply:
As we just discussed, Nick, wondering if we might just get away with simpler logic anyway?

``` 
- if case_details.third_party?
   = render partial 'third_party_requester_details'
```
... because "third_party" specifically means "third party requester" for Offender SAR cases and is slightly independent of the recipient - recipient will either be subject or requester if "third_party" is true but I don't _think_ we need this part of the page to check

And then in the next section about where to send the data, we can just use 

```
- if case_details.third_party_recipient?
          = render partial: 'cases/offender_sar/third_party_recipient_details'
```